src/app/pages/account/update/index.js

```diff
@@ -0,0 +1,37 @@
+/* ============
+ * Account Update Page
+ * ============
+ *
+ * Page where the user can update the account information
+ */
+import accountService from '../../../services/account'
+import store from '../../../store/modules/account'
+
+
+export default {
+  computed: {
+    account: {
+      get () {
+        return store.state;
+      },
+      set (account) {
+        accountService.update(account);
```

Comment:
Don't do anything here. No need to update the account each time the value has been update.
Just do this:
``` JavaScript
set (account) {
  //
}
```